README.md

```diff
@@ -37,18 +36,16 @@ $ make build
 
 Have a look at CONTRIBUTING.md for more developer information.
 
-For implementation details, see [RFC-11](https://github.com/kubewarden/rfc/blob/main/rfc/0011-audit-checks.md), 
+For implementation details, see [RFC-11](https://github.com/kubewarden/rfc/blob/main/rfc/0011-audit-checks.md),
 [RFC-12](https://github.com/kubewarden/rfc/blob/main/rfc/0012-policy-report.md).
```

Comment:
We should go back and update the RFCs to reflect the changes done.

Reply:
Tracked: https://github.com/kubewarden/audit-scanner/issues/209

Reply:
thanks!